src/test/scala/com/advancedtelematic/director/http/DeviceResourceSpec.scala

```diff
@@ -336,6 +339,13 @@ trait DeviceResourceSpec extends DirectorSpec with KeyGenerators with DefaultPat
     updateManifestOk(device, deviceManifestTarget)
 
     coreClient.getReport(updateId) shouldBe Seq(operation)
+    fetchTargetsFor(device).signed.targets.isEmpty shouldBe true
+
+    // schedule and confirm again
+    schedule(device, targets, updateId)
+    fetchTargetsFor(device).signed.targets.isEmpty shouldBe false
```

Comment:
this should be `fetchTargetsFor(device).signed.targets shouldBe empty` for a better message on failure